packages/@react-spectrum/s2/src/Menu.tsx

```diff
@@ -157,6 +157,7 @@ export let menuitem = style({
   },
   paddingBottom: '--labelPadding',
   backgroundColor: { // TODO: revisit color when I have access to dev mode again
+    // @ts-expect-error Expression produces a union type that is too complex to represent
```

Comment:
This will be fixed by [chore: Update typescript to 5.8](https://github.com/adobe/react-spectrum/pull/7888#top)